docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:14.1-alpine
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - '5432:5432'
+    volumes: 
+      - db:/var/lib/postgresql/data
+  pgadmin:
+    image: dpage/pgadmin4
+    restart: always
+    environment:
+      PGADMIN_DEFAULT_EMAIL: admin@admin.com
+      PGADMIN_DEFAULT_PASSWORD: root
```

Comment:
consider reading these from the `.env` file - we don't have to do this right now in this PR, but sooner or later we should change it

Reply:
also once this is done we'll have to use different credentials as these are already committed and should be treated as leaked